src/graphics/graphics_device.js

```diff
@@ -912,10 +934,20 @@ pc.extend(pc, function () {
             }
 
             if (primitive.indexed) {
-                gl.drawElements(this.glPrimitive[primitive.type],
-                                primitive.count,
-                                this.indexBuffer.glFormat,
-                                primitive.base * 2);
+                if (numInstances > 1) {
+                    this.extInstancing.drawElementsInstancedANGLE(this.glPrimitive[primitive.type],
+                                                                  primitive.count,
+                                                                  this.indexBuffer.glFormat,
+                                                                  primitive.base * 2,
+                                                                  numInstances);
+                    this.boundBuffer = [];
+                    this.attributesInvalidated = true;
+                } else {
+                    gl.drawElements(this.glPrimitive[primitive.type],
+                                    primitive.count,
+                                    this.indexBuffer.glFormat,
+                                    primitive.base * 2);
+                }
             } else {
                 gl.drawArrays(this.glPrimitive[primitive.type],
                               primitive.base,
```

Comment:
The FBX -> JSON tools only generates indexed, but you can create pc.Mesh objects that are unindexed procedurally.